alkira/resource_alkira_internet_applications.go

```diff
@@ -162,6 +162,12 @@ func resourceAlkiraInternetApplication() *schema.Resource {
 				},
 				Optional: true,
 			},
+			"ilb_credential_id": {
+				Description: "Credential ID of the ILB Target.  This can only be used if the associated connector is an AWS_VPC",
```

Comment:
where this credential comes from? It's AWS credential? or which credential. The comment is confusing.